tests/network/example_fuji_network.go

```diff
@@ -0,0 +1,252 @@
+package network
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"strings"
+	"time"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/subnet-evm/core/types"
+	"github.com/ava-labs/subnet-evm/ethclient"
+	"github.com/ava-labs/subnet-evm/interfaces"
+	teleportermessenger "github.com/ava-labs/teleporter/abi-bindings/go/Teleporter/TeleporterMessenger"
+	"github.com/ava-labs/teleporter/tests/utils"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/log"
+	. "github.com/onsi/gomega"
+)
+
+var _ Network = &FujiNetwork{}
+
+// Amplify, Bulletin, Conduit subnet constants
```

Comment:
Can we add a note here somewhere that the tests will not currently works against these networks due to incompatibility with the `subnet-evm` version deployed on them?

We will update in #132 either way.

Reply:
added [here](https://github.com/ava-labs/teleporter/pull/93/files#diff-73cd34956e9a9bb0832f207ea9a0c60067ea99aeec1509dc291f1de930606adaR24-R25)